js/runner.go

```diff
@@ -72,8 +73,9 @@ type Runner struct {
 	ActualResolver netext.MultiResolver
 	RPSLimit       *rate.Limiter
 
-	console   *console
-	setupData []byte
+	console           *console
+	setupData         []byte
+	nameToCertWarning sync.Once
```

Comment:
I think should be better to have it as a global variable, otherwise, every new runner (e.g. in tests) re-prints the message.

Reply:
done